### PR TITLE
Added placeholder support for text fields

### PR DESCRIPTION
**Pull request by eddie-xavi**

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -13,6 +13,7 @@ export interface Field extends NameLabelPair {
   values?: NameLabelPair[];
   defaultValue?: any;
   [x: string]: any;
+  placeholder?: string | '';
```
Comment (jakeboone02) [inline src/types.ts]: This should just be `string`.  The `| ''` is redundant.

**Review by jakeboone02 — CHANGES_REQUESTED** [review]
Could you also update the documentation in README.md?

**Comment by eddie-xavi** [comment]
> Could you also update the documentation in README.md?

Updated!

**Review by jakeboone02 — APPROVED** [review]

**Comment by jakeboone02** [comment]
This feature was shipped in [release 3.8.4](https://github.com/sapientglobalmarkets/react-querybuilder/releases/tag/v3.8.4).  Thanks for your contribution, @eddie-xavi!

**Pull request merged**